### ISSUE TITLE
Server don`t start after update 3.1.0

### ISSUE THREAD

**Issue by Nidre** [natanfudge/Not-Enough-Crashes#33]
Server don`t start after update 3.1.0
[latest.log](https://github.com/natanfudge/Not-Enough-Crashes/files/5894690/latest.log)

**Comment by Blobanium** [comment]
the same thing seems to be happening with the client as well, seems like its something to do with the mixin probably
![Capture](https://user-images.githubusercontent.com/45613796/106311563-356deb80-621a-11eb-86cd-18356e1647d9.PNG)

**Comment by natanfudge** [comment]
Thanks for reporting
Please use 3.0.0 for now

**Comment by inikishev** [comment]
same issue with the same mod list https://filebin.net/n1g6bujgdzhhh3zp i posted in other issue  but after that I updated rswires to latest version (not server but launching minecraft). 3.0.0 works
```
[21:57:23] [main/INFO]: Loading for game Minecraft 1.16.5
[21:57:24] [ForkJoinPool-1-worker-1/WARN]: The mod "aquarius" contains invalid entries in its mod json:
- Unsupported root entry "licence" at line 9 column 12
[21:57:24] [ForkJoinPool-1-worker-5/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-3/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-7/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-4/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-3/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-6/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-2/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-1/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-5/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-6/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [ForkJoinPool-1-worker-6/WARN]: The mod "qcommon-croco" contains invalid entries in its mod json:
- Unsupported root entry "links" at line 10 column 10
[21:57:24] [ForkJoinPool-1-worker-4/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[21:57:24] [main/WARN]: Warnings were found! 
 - Conflicting versions found for fabric-api-base: used 0.2.0+ab87788d3a, also found 0.2.0+daa38b3d7d
 - Conflicting versions found for terraform-config-api-v1: used 1.0.0+build.4, also found 1.0.0+build.1
 - Conflicting versions found for terraform-surfaces-api-v1: used 1.0.0+build.4, also found 1.0.0+build.1
 - Conflicting versions found for terraform-biome-builder-api-v1: used 1.0.0+build.4, also found 1.0.0+build.1
 - Mod 'Beenfo' (beenfo) recommends version 1.1.7 or later of mod modupdater, which is missing!
	 - You must install version 1.1.7 or later of modupdater.
 - Conflicting versions found for terraform-overworld-biome-extensions-api-v1: used 1.0.0+build.4, also found 1.0.0+build.1
[21:57:24] [main/WARN]: Mod `{}` ({}) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'
[21:57:24] [main/WARN]: Mod `{}` ({}) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'
[21:57:24] [main/INFO]: [FabricLoader] Loading 241 mods: fermion@2.9.218, roughlyenoughitems-api@5.9.1, fermion-orientation@1.2.218, simple_utilities@2.1.0, nenchantfix@3.0.0, fabricloader@0.11.1, crowdin-translate@1.2, eymbra@1.0.0, fabriczero@0.2.1, prefab@1.0.2, fabric-dimensions-v1@2.0.2+ca58154a7d, cloth-durability-bar-api-v1@1.4.9, java@8, rad@1.0.1, betternether@5.0.7, fabric-game-rule-api-v1@1.0.6+ca58154a7d, agape_space@0.0.6, adventurez@1.1.5, fermion-modkeys@1.10.218, fabric-api-base@0.2.0+ab87788d3a, astromine-discoveries@1.11.5+fabric-1.16.5, special-circumstances@1.8.218, yungsapi@1.16.4-Fabric-1, org_joml_joml@1.10.0, randompatches@2.1.9-fabric, maybe-data@1.0.0-1.16.2, world_blender@1.16.4-2.0.1, packages@1.1.1, astromine@1.11.5+fabric-1.16.5, terraform-dirt-api-v1@1.1.0+build.4, astromine-transportations@1.11.5+fabric-1.16.5, json-model-extensions@1.19.172, astromine-foundations@1.11.5+fabric-1.16.5, terraform-wood-api-v1@1.0.1, org_jetbrains_kotlin_kotlin-stdlib-jdk7@1.4.21, multi_item_lin@1.0.0, soulcapture@1.0.2, fabric-networking-v0@0.3.2+ca58154a7d, jargm@1.2.0, tk_valoeghese_zoesteriaconfig@1.3.2, repurposed_structures@1.16.4-1.7.3, modmenu@1.14.15, phosphor@0.7.0+build.10, cloth-utils-v1@1.4.5, launchersmod@2.2.0, org_jetbrains_kotlin_kotlin-stdlib-jdk8@1.4.21, roughlyenoughitems-runtime@5.9.1, battletowers@3.3.2-1.16.4, frex@4.5.201, structure-helpers@3.0.0, voyager@1.0.0, fabric-crash-report-info-v1@0.1.3+ca58154a7d, oro_config@unspecified, mostructures@1.0.1+mc.1.16.4, overpowered_mending@2.2.2, fabric-key-binding-api-v1@1.0.2+ca58154a7d, disable_custom_worlds_advice@1.2, programmerartinjector@1.2.1, libblockattributes_fluids@0.8.5, staticdata@1.1.2, magna@1.4.3-1.16.2, exotic-matter@2.11.379, bedrockbreaker@1.0, fabric-networking-api-v1@1.0.1+ca58154a7d, bettermineshafts@1.16.4-1.0, geckolib3@3.0.13, org_jetbrains_annotations@20.0.0, terraform-shapes-api-v1@1.0.1, alaskanativecraft@1.0.2, terrestria@2.1.6, gbfabrictools@1.3.2+1.16.4, wi_zoom@1.3-MC1.16.5, flamboyant@1.16.4-1.0.1, better_wandering_trader@1.6.2, libaddict@0.2.1, fabric-structure-api-v1@1.1.4+ca58154a7d, lithium@0.6.1, fabric-object-builder-api-v1@1.9.3+ca58154a7d, hctm-base@3.5.1, harvest_scythes@1.0.9, libblockattributes@0.8.4, terraform-config-api-v1@1.0.0+build.4, infinitywaterbucket@1.0.3.2-compat-mc1.16.4-fabric, fabric-rendering-data-attachment-v1@0.1.5+ca58154a7d, extradoors@1.8.2-1.16.2, terraform-surfaces-api-v1@1.0.0+build.4, reach-entity-attributes@1.0.1, environmentalcreepers@1.6.1, fabric-lifecycle-events-v1@1.2.1+ca58154a7d, cardinal-components-util@2.7.10, fabric-loot-tables-v1@1.0.2+ca58154a7d, opchanter@1.16.5-fabric, crumbs@1.0.6, dynamicfps@2.0.1, blame@1.16.4-2.4.1, autoconfigtoml@1.0.0-fabric, libblockattributes_core@0.8.5, fastchest@1.0, terraform-biome-builder-api-v1@1.0.0+build.4, minecraft@1.16.5, fabric-command-api-v1@1.0.10+ca58154a7d, jankson@3.0.1+j1.2.0, classicalart@1.0.0, gildednetherite@1.6.0, fabric-renderer-indigo@0.4.4+ca58154a7d, org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm@1.4.2, libgui@3.2.2+1.16.3, fabric-entity-events-v1@1.0.3+ca58154a7d, fabric-rendering-v0@1.1.2+ca58154a7d, croptopia@1.2.0, fabric-rendering-v1@1.5.1+ca58154a7d, polymorph@0.0.8-1.16.4, quitconfirm@1.2-SNAPSHOT, autoconfig1u@3.3.1, fabric-tag-extensions-v0@1.1.1+ca58154a7d, blue_endless_jankson@1.2.0, xb@1.6.171, byg@1.1.3, cloth-datagen-api-v1@1.4.9, cinderscapes@1.3.2, fabric-events-lifecycle-v0@0.2.1+ca58154a7d, aquarius@1.0.3, phonos@0.1.1-beta, fabric-tree-chopper@0.6.1, betterend@0.8.8-beta, team_reborn_energy@0.1.1, fabric-keybindings-v0@0.2.1+ca58154a7d, notenoughcrashes@3.1.0, artifice@0.13.2+1.16.3, beenfo@1.16.4-fabric0.25.1-1.3, silky_spawners@2.1.0+build.2-1.16, nostrip@1.2.0-1.16.2+, staffofbuilding@1.2.4-1.16.2, glassential@1.16.4-1.1.5, darkness@1.6.58, xaeroworldmap@1.11.10, auto_planter@16.1-fabric, piglib@1.0.1, canvas@1.0.1267, fiber@0.23.0-2, fabrication@1.3.1, cardinal-components-level@2.7.10, com_moandjiezana_toml_toml4j@0.7.2, org_jetbrains_kotlin_kotlin-stdlib@1.4.21, colorfulcolumns@1.0.0, fabric-blockrenderlayer-v1@1.1.5+ca58154a7d, libblockattributes_items@0.8.5, gameinfo@2.0.1 1.16.4, fabric@0.30.0+1.16, netherite_plus@1.3.10-1.16.5, wrenchable@1.0.0+1.16.1, fabric-screen-handler-api-v1@1.1.1+ca58154a7d, columns@1.1.1, fabric-tool-attribute-api-v1@1.2.6+ca58154a7d, terraform-tree-api-v1@1.0.1+build.4, angerable-patch@1.1.0-1.16.4, autofish@0.9.2, cloth-client-events-v0@1.4.5, blockus@2.0.2+1.16.4, fabric-events-interaction-v0@0.4.2+ca58154a7d, mob_proof_crystals@1.1.0, fabric-language-kotlin@1.4.21+build.1, enchantmentdescriptions@1.0.0, patchouli@1.16.4-48-FABRIC, cullleaves@2.0.0, cloth-config2@4.8.3, org_jetbrains_kotlinx_kotlinx-coroutines-jdk8@1.4.2, advancednbttooltip@1.3.0+build.2-1.16.4, fabric-resource-loader-v0@0.4.2+ca58154a7d, org_jetbrains_kotlin_kotlin-reflect@1.4.21, cardinal-components-entity@2.7.10, architectury@1.3.83, computercraft@1.91.2, astromine-technologies@1.11.5+fabric-1.16.5, fabric-registry-sync-v0@0.7.4+ca58154a7d, fabric-commands-v0@0.2.2+ca58154a7d, cardinal-components-base@2.7.10, rswires@4.10.1, fabric-mining-levels-v0@0.1.3+ca58154a7d, fabric-models-v0@0.2.1+ca58154a7d, fabric-item-groups-v0@0.2.3+ca58154a7d, shulkerboxtooltip@2.3.4+1.16.4, fabric-renderer-api-v1@0.4.1+ca58154a7d, inventorysorter@1.7.6-1.16, cardinal-components-block@2.7.10, supernaturalcrops@1.0.2, biomemakeover@1.16.4-1.2.9, cavebiomes@0.5.1, simplevoidworld@4.0.0, automated_crafting@1.3.20+MC1.16.2-1.16.5, blade@2.0.0, fabric-containers-v0@0.1.10+ca58154a7d, accurateblockplacement@1.0.8, redbits@1.7.0, plantinajar@1.8.1, appliedenergistics2@8.1.0-alpha.13, vanillatweaks@1.16.4-1.0.6, campanion@1.3.2, fabric-textures-v0@1.0.6+ca58154a7d, cardinal-components-item@2.7.10, com_electronwill_night-config_core@3.6.3, qcommon-cfg-core@3.2.0, fabric-rendering-fluids-v1@0.1.13+ca58154a7d, traverse@3.3.1, cardinal-components-chunk@2.7.10, fabric-renderer-registries-v1@2.2.1+ca58154a7d, fabric-particles-v1@0.2.4+ca58154a7d, recipeconditions@0.4.0, artofalchemy@1.0.0-rc4+1.16.4, roughlyenoughitems-default-plugin@5.9.1, fabric-networking-blockentity-v0@0.2.8+ca58154a7d, exotic-art-core@1.0.379, cloth-basic-math@0.5.1, qcommon-croco@2.1.1, javax_vecmath_vecmath@1.5.2, frex-events@1.3.13, fatxporbs@0.0.9, static-content@1.0.1-1.16.2, ward_blocks@1.0.20, fabric-screen-api-v1@1.0.0+c045166c7d, microdurability@0.3.0+1.16.3, verticalredstone@1.2.0, com_electronwill_night-config_toml@3.6.3, fiber2cloth@3.1.0, eer@1.0.0, roughlyenoughitems@5.9.1, fabric-item-api-v1@1.2.1+ca58154a7d, kyrptconfig@1.1.6-1.16, biome_locator@1.0.0, automotons@1.0.1, terraform-overworld-biome-extensions-api-v1@1.0.0+build.4, morepaintings@1.0.2, fabric-content-registries-v0@0.2.1+ca58154a7d, cardinal-components-world@2.7.10, fabric-biome-api-v1@3.1.1+ca58154a7d, betterf3@1.1.2, libstructure@1.5, minegate-moreblocks@1.0.2, astromine-core@1.11.5+fabric-1.16.5, fabric-object-builders-v0@0.7.2+ca58154a7d, fermion-varia@2.6.218
[21:57:24] [main/WARN]: Mod `oro_config` (unspecified) does not respect SemVer - comparison support is limited.
[21:57:24] [main/WARN]: Mod `infinitywaterbucket` (1.0.3.2-compat-mc1.16.4-fabric) uses more dot-separated version components than SemVer allows; support for this is currently not guaranteed.
[21:57:24] [main/WARN]: Mod `gameinfo` (2.0.1 1.16.4) does not respect SemVer - comparison support is limited.
[21:57:31] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/nkshv/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.8.2+build.24/sponge-mixin-0.8.2+build.24.jar Service=Knot/Fabric Env=CLIENT
[21:57:31] [main/INFO]: Loaded configuration file for Lithium: 66 options available, 0 override(s) found
[21:57:31] [main/WARN]: Reference map 'Croptopia-1.16.4-FABRIC-refmap.json' for Croptopia.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:31] [main/WARN]: Reference map 'gilded-netherite-fabric-refmap.json' for gildednetherite.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:31] [main/INFO]: [ExtraDoors] Running with lithium, disabling pathing mixin
[21:57:31] [main/WARN]: Reference map 'soulcapture-refmap.json' for soulcapture.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:31] [main/WARN]: Reference map 'static-content-refmap.json' for static-content.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:31] [main/WARN]: FabricZero is in it's end of life, please consider stopping using it!
[21:57:31] [main/INFO]: FabricZero: Loaded!
[21:57:32] [main/WARN]: Reference map 'astromine-transportations-refmap.json' for astromine-transportations.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:32] [main/WARN]: Reference map 'blade-2.0.0+fabric-1.16.3-refmap.json' for blade.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:32] [main/INFO]: [Fabrication] Reloading configs...
[21:57:32] [main/INFO]: [Fabrication] Loading of features.ini done in 10.81 ms with 0 warnings
[21:57:32] [main/WARN]: Overriding InjectionInfo for @Inject with com.unascribed.fabrication.support.injection.FailsoftCallbackInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo)
[21:57:32] [main/WARN]: Overriding InjectionInfo for @ModifyArg with com.unascribed.fabrication.support.injection.FailsoftModifyArgInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyArgInjectionInfo)
[21:57:32] [main/WARN]: Overriding InjectionInfo for @ModifyArgs with com.unascribed.fabrication.support.injection.FailsoftModifyArgsInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyArgsInjectionInfo)
[21:57:32] [main/WARN]: Overriding InjectionInfo for @Redirect with com.unascribed.fabrication.support.injection.FailsoftRedirectInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.RedirectInjectionInfo)
[21:57:32] [main/WARN]: Overriding InjectionInfo for @ModifyVariable with com.unascribed.fabrication.support.injection.FailsoftModifyVariableInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyVariableInjectionInfo)
[21:57:32] [main/WARN]: Overriding InjectionInfo for @ModifyConstant with com.unascribed.fabrication.support.injection.FailsoftModifyConstantInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyConstantInjectionInfo)
[21:57:32] [main/WARN]: Reference map 'flamboyant-refmap.json' for flamboyant.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:32] [main/WARN]: Reference map 'wrenchable-refmap.json' for wrenchable.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:32] [main/WARN]: Reference map 'astromine-technologies-refmap.json' for astromine-technologies.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:32] [main/WARN]: Reference map 'astromine-foundations-refmap.json' for astromine-foundations.mixins.json could not be read. If this is a development environment you can ignore this message
[21:57:32] [main/WARN]: @Mixin target net/minecraft/class_3962$class_3963 is public in lithium.mixins.json:alloc.composter.ComposterMixin$ComposterBlockComposterInventoryMixin and should be specified in value
[21:57:33] [main/WARN]: @Mixin target net/minecraft/class_3778$class_4182 is public in blame.mixins.json:StructurePoolBasedGeneratorStructurePoolGeneratorMixin and should be specified in value
[21:57:33] [main/WARN]: @Mixin target net/minecraft/class_3898$class_3208 is public in fabric-networking-api-v1.mixins.json:accessor.EntityTrackerAccessor and should be specified in value
[21:57:34] [main/INFO]: Instancing error handler class com.unascribed.fabrication.support.MixinErrorHandler
[21:57:34] [main/FATAL]: Mixin apply failed notenoughcrashes.mixins.json:MixinCrashReport -> net.minecraft.class_128: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method generateWittyComment in notenoughcrashes.mixins.json:MixinCrashReport was not located in the target class net.minecraft.class_128. Using refmap notenoughcrashes-common-refmap.json
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow method generateWittyComment in notenoughcrashes.mixins.json:MixinCrashReport was not located in the target class net.minecraft.class_128. Using refmap notenoughcrashes-common-refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachSpecialMethod(MixinPreProcessorStandard.java:423) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachShadowMethod(MixinPreProcessorStandard.java:399) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachMethods(MixinPreProcessorStandard.java:327) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:286) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:264) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1272) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:287) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) [sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) [sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) [sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) [sponge-mixin-0.8.2+build.24.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) [fabric-loader-0.11.1.jar:0.8.2+build.24+unknown-b24.git-unknown]
	at com.fox2code.fabriczero.FabricZeroTransformerHook.transformClassBytes(FabricZeroTransformerHook.java:41) [fabriczero-0.2.1.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157) [fabric-loader-0.11.1.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150) [fabric-loader-0.11.1.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) [?:1.8.0_271]
	at net.minecraft.client.main.Main.main(Main.java:135) [intermediary-fabric-loader-0.11.1-1.16.5.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_271]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_271]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_271]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_271]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.1.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139) [fabric-loader-0.11.1.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27) [fabric-loader-0.11.1.jar:?]

```

**Comment by triphora** [comment]
Ironic.

**Comment by natanfudge** [comment]
Fixed as of 3.1.1
